demo/src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-server.port=8080
+server.port=8081
```

Comment:
pretpostavljam da ovo radis zato sto ti je port 8080 na racunaru bio zauzet. Alikaciju u kontejneru i dalje mozes pokretati na portu 8080 i samo promeniti port za pristup kontejneru. Nije greska, samo ti pisem da sledeci put ne bi gubila vreme na promene portova i u aplikaciji i ponovno build-ovanje kontejnera :)

Reply:
Dada, dole sam vam to spomenula u komentaru, u jednom trenutku mi nije radilo pa sam uzela i izmenila sve portove u nadi da je to greska, medjutim to nije bio problem ali sam zaboravila da vratim na staro :D